cli.js

```diff
@@ -0,0 +1,14 @@
+const { Command } = require('commander');
+const fs = require('fs');
+const exportModule = require('./exportModule.js');
+
+const program = new Command();
+program.version('0.0.1');
+function data() {
+  program.option('--elmJSON <file>', 'elm json to turn to synthea module').parse(process.argv);
```

Comment:
We could use `requiredOption` here so the CLI tells the user that they need to provide this argument

```suggestion
  program.requiredOption('--elmJSON <file>', 'elm json to turn to synthea module').parse(process.argv);
```